tasks/rosetta-code/find-unimplemented-tasks/src/lib.rs

```diff
@@ -102,9 +102,7 @@ impl Iterator for Category {
     type Item = Vec<Task>;
 
     fn next(&mut self) -> Option<Self::Item> {
-        if self.continue_params.is_none() {
-            return None;
-        }
+        self.continue_params.as_ref()?;
 
         query_api(&self.name, self.continue_params.as_ref().unwrap())
```

Comment:
Should we fix it here, too?

Reply:
Let me know if you meant something else!